docs/setup.md

```diff
@@ -1,10 +1,69 @@
 # Local Development Environment
 
-## Requirements
+__NOTE: Documentation assumes you are using MacOS with at least 8GB of memory on an Intel, M1, or M2 processor__
+
+:clap: Thank you for considering to contribute to the YaleSites Platform!
+
+## Requirements & Setup
+
+1. [Contact YaleSites](mailto:yalesites@yale.edu?subject=Developer%20Access%20Request) for access to resources such as Pantheon, Figma designs, etc.
+2. [GitHub](#github)
+   1. SSH authorization key
+   2. [Github Package Personal Access Token](#package-personal-access-token) in environment variable `YALESITES_BUILD_TOKEN`
+3. [NVM](#additional-tools)
+4. [Node.js (>= 8.0, < 11.0)](#additional-tools)
+5. [Composer](#additional-tools): Version 2.x.
+6. [Terminus](#terminus): Machine auth token
+7. [Lando](#lando)
+8. Docker: Use the version Lando wants to install and increase memory resources to at least 3GB memory and 3 CPUs if possible
+9. [Project Files](#project-setup-script)
+    ```bash
+    # Set up local development version of repo
+    git clone git@github.com:yalesites-org/yalesites-project
+    cd yalesites-project
+    npm run setup
+    ```
+10. [Project Commands](#project-commands)
+
+## Repository Links
+
+Repositories used to make the platform:
+
+* [Yalesites Project](https://github.com/yalesites-org/yalesites-project): Drupal Platform Site (This repo)
+   * [Yalsites Project Profile modules](https://github.com/yalesites-org/yalesites-project/tree/develop/web/profiles/custom/yalesites_profile/modules/custom): Each subdirectory should have a README describing the function
+* [Atomic Theme](https://github.com/yalesites-org/atomic): Atomic Drupal Theme bridging the Drupal site and the component library
+* [Component Library Twig](https://github.com/yalesites-org/component-library-twig): Component Library
+   * [Deployed Storybook](https://yalesites-org.github.io/component-library-twig)
+* [Tokens](https://github.com/yalesites-org/tokens): Style tokens from Figma used to drive design of the platform and components
+* [ESLint Config and Other Formatting](https://github.com/yalesites-org/eslint-config-and-other-formatting): Reusable Linting/Formatting included in the project
+
+## Requirement Details
+
+### GitHub
+
+While the project can be checked out and run locally without it, one must setup an SSH key on their development machine if they wish to push code on any of the yalesites repositories.
```

Comment:
This should probably say "cloned" instead of "checked out"

Reply:
Good catch.  Done!